components/content-search/index.js

```diff
@@ -357,6 +359,8 @@ const ContentSearch = ({
 					onBlur={() => {
 						setIsFocused(false);
 					}}
+					size="compact"
```

Comment:
Actually I think for us here the not compact size is better

Reply:
Removed